CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Added `adls_bulk_upload` task function to `task_utils.py`
+- Added `test_adls_bulk_upload` test function to `test_task_utils.py`
```

Comment:
Information about tests is not important for the changelog. We are storing only information about the viadot code.

Reply:
removed!